python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -1093,7 +1093,8 @@ def _outputZeropoints(self, camera, zptCat, visitCat, offsets, bands,
         # ccds.
         cannot_compute = fgcm.fgcmUtilities.zpFlagDict['CANNOT_COMPUTE_ZEROPOINT']
         selected = (((zptCat['fgcmFlag'] & cannot_compute) == 0)
-                    & (zptCat['fgcmZptVar'] > 0.0))
+                    & (zptCat['fgcmZptVar'] > 0.0)
+                    & (zptCat['fgcmZpt'] > -9999.0))
```

Comment:
As is, I had no idea why this would do the trick.  Having perused the code, I see this is the value hardcoded for `illegalValue` in the `configDict` defined `utilities.py`.  Would it be possible to get this number from said dict so that you don't risk divergence if one or the other changes?

I'm also not sure what qualifies something as "illegal".  Is there a note somewhere about this that I've missed?

Reply:
So this is necessary because of a bug/misfeature in fgcmcal that it's not flagging certain outputs in a way that it should, so this is a workaround.  I can't get it from the config dict because that's from a different task (sigh).  I could instead add it as an ALL_CAPS_CONSTANT, I'll do that.  (This is an internal fgcm signifier because I didn't want nans floating around).